### PR TITLE
Add Spotify Docker image

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

File: spotify/Makefile [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .PHONY: run dev
 run dev:
-	python3 main.py
\ No newline at end of file
+	python3 main.py
+
+.PHONY: docker-build
+docker-build:
+	docker build -t openpodcast/spotify-cron .
```
Comment (woolfg) [inline spotify/Makefile:7]: why is the name cron? currently there is no cron, do we add it later?

**Review by woolfg — APPROVED** [review]
besides the missing cron part, it looks good!

File: spotify/Makefile (hunk 1 above)
Comment (mre) [reply spotify/Makefile:7]: Yes, I'll add that part. However I'm still not sure about the image name.

Comment (woolfg) [reply spotify/Makefile:7]: maybe just spotify-connector?

Comment (mre) [reply spotify/Makefile]: Check. Changed it to `spotify-connector`.

**Pull request merged**